motor/providers/microsoft/roles.go

```diff
@@ -0,0 +1,45 @@
+package microsoft
+
+var DefaultRoles = []string{
```

Comment:
I think we w need to be a bit more explicit. Those roles only apply to msgraph

Reply:
I added a comment thats only msgraph stuff. We can rework this in the future, once we start getting the token's permissions as they are, instead of using default roles.